docs/TeX/index.md

```diff
@@ -864,3 +864,32 @@ My private examples:
 - by tikz based on filtered paths
 
 refer to [Can we make a love heart with LaTeX?](https://tex.stackexchange.com/questions/139733/can-we-make-a-love-heart-with-latex)
+
+	
+## Change section title without altering the cleveref cross-referencing system
+Suppose you (1) want the section title to be "Good Title 1", "Good Title 2", etc., and (2) you want cross-reference two sections at the same time as "Good Titles 1 and 2" instead of "Good Title 1 and Good Title 2". Although one can do it mannually, the author is lazy and would like the cross-referencing system to do it automatically.
```

Comment:
you can add your name and date after the section title. BTW,  in Markdown, an empty line is preferred between different elements (title, content, code block, etc.), so you can leave an empty line after the title